### PR TITLE
Doc: Improve docstrings on UIWrapper

### PR DESCRIPTION
**Pull request by kitchoi**
This PR fixes a few missing "Returns" block in the `UIWrapper` docstrings (part of the public API) and some drive-by language improvements.
Purely documentation change, i.e. no behavioural change.

### CODE REVIEW

File: traitsui/testing/tester/ui_wrapper.py [hunk 1]
```diff
@@ -132,14 +132,19 @@ def locate(self, location):
         Parameters
         ----------
         location : any
-            Any location type supported by the current target.
+            An instance of a location type supported by the current target.
+
```
Comment (rahulporuri) [inline traitsui/testing/tester/ui_wrapper.py:136]: nit: do you want to add an example here like you did with the docstring for interaction?

**Review by rahulporuri — APPROVED** [review]
LGTM

File: traitsui/testing/tester/ui_wrapper.py (hunk 1 above)
Comment (kitchoi) [reply traitsui/testing/tester/ui_wrapper.py:136]: Ah yes, that's a good idea.

**Comment by rahulporuri** [comment]
Still LGTM

**Pull request merged**